src/main/java/lotto/LottoProgram.java

```diff
@@ -0,0 +1,23 @@
+package lotto;
+
+import static lotto.view.InputView.inputPurchaseAmount;
+import static lotto.view.InputView.inputWinningNumbers;
+import static lotto.view.ResultView.printLottosCount;
+import static lotto.view.ResultView.printReturnRate;
+import static lotto.view.ResultView.printWinningResult;
+
+public class LottoProgram {
+
+    public void run() {
+        int purchaseAmount = inputPurchaseAmount();
+
+        Lottos lottos = LottoMachine.createLottos(
+            LottoStore.sellLotto(purchaseAmount), new RandomLottoGenerator());
```

Comment:
```suggestion
        Lottos lottos = Lottos::생성해줘(4개, 숫자생성기interface);
```

생성에 대한 책임을 Lottos에게 주는 관점은 어떤가요? 😄

Reply:
생성에 대한 책임을 Lottos에 줌으로써 얻는 이득은 무엇일까요?

현실 세계로 보면 LottoMachine이 로또들을 만들어 주는게 자연스럽다고 생각해서 LottoMachine이란 클래스를 만들었습니다.
하지만 코드를 작성하다보면 LottoMachine과 같이 static Method를 하나 가지고 있는 클래스를 만들게 되는데요 !
이렇게 까지 클래스를 만들어서 분리를 해야할까? 라는 생각을 언뜻하긴 했습니다만, 
정작 이럴 때 클래스를 나누었을 때 이득은 무엇일까? Lottos가 생성에 대한 책임을 가지면 어떤 이득이 있을까? 에 대한 물음에는 정확히 답하지 못하겠습니다.

클래스를 나눔으로써 여러개로 나누어진 클래스에 대한 관리에 대한 피로때문일까요?
저는 LottoMachine이란 클래스를 알지만 타인이 해당 코드를 보았을 때 피로감이 생길까요?
아니면 객체에 대한 생성은 클래스 자신이 책임져야 하는게 자연스러워서 일까요?

멘토님 생각이 궁금합니다 !